zombies.js

```diff
@@ -7,7 +7,14 @@
  * @param {string} name     The item's name.
  * @property {string} name
  */
-
+  class Item {
+    constructor(name) {
+      this._name = name
+    }
+    get name(){
```

Comment:
Each getter should have a matching setter. Please update all your getters in the file to have a matching setter.